Docker/python-environment/Dockerfile

```
@@ -20,5 +21,17 @@ RUN useradd -m -u $UID -s /bin/bash $UNAME
 RUN usermod --groups sudo -a $UNAME
 RUN echo "$UNAME  ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
+# Virtual environments are required, because later versions of python do not allow installing packages globally
+# PYTHON_ENVIRONMENT_PYTHON3 has to be used to when using python3 in docker images to avoid using the system python3
+ENV PYTHON_ENVIRONMENT_PYTHON3=/home/$UNAME/venv/bin/python3
+ADD python_warning.sh /home/$UNAME/python_warning.sh
```

Hadolint

[error] Use COPY instead of ADD for files and folders

(DL3020)